src/css/project-tabs/QRtab.css

```diff
@@ -26,13 +26,37 @@
     text-align: center;
 }
 p{
-    font-size: 18px;
+    /*font-size: 18px;*/
 }
-ol{
+.PC_ordered_list{
     font-size: 18px;
     text-align: justify;
 }
 .clear{
     clear: both;
 }
 
+.Mobile_QR{
+    text-align: center;
+}
+
+.Mobile_QRTab_Title{
+    text-align: center;
+    /*left: 100%;*/
+    font-size: 20px;
+}
+.Mobile_instructionDiv{
+    width: 100%;
+    /*display: block;*/
+}
+.mobile_ordered_list{
```

Comment:
Make the naming scheme more consistent with what's already being done in the project. I suggest doing everything in lower case and using underscores, or whatever styling is already being used in the other css files.